test/builtin_dataset_mocks.py

```diff
@@ -1438,6 +1438,97 @@ def stanford_cars(info, root, config):
     return num_samples
 
 
+@register_mock
+def places365(info, root, config):
+    num_samples = {"train": 5, "test": 4, "val": 3}[config.split]
+    categories = ["alley", "coffee_shop", "field/wild", "subway_station/platform"]
```

Comment:
Could we use the last category here to catch potential issues with zero / one based indexing of the labels?

```suggestion
    categories = ["alley", "field/wild", "subway_station/platform", "zen_garden"]
```